WordPress/Classes/ViewRelated/Comments/CommentDetailInfoViewController.swift

```diff
@@ -52,7 +53,10 @@ final class CommentDetailInfoViewController: UIViewController {
 
     private func setPreferredContentSize() {
         tableView.layoutIfNeeded()
-        preferredContentSize = tableView.contentSize
+
+        if UIDevice.isPad() {
```

Comment:
What do you think changing this condition to something like `navigationController?.modalPresentationStyle == .popover`?

Reply:
Oh, forgot to remove it. It's safe to not have any condition here. `preferredContentSize` will only be used by the presentation controllers that need it, like the popover. Updated.